core/network/src/test/kotlin/com/google/samples/apps/nowinandroid/core/network/fake/FakeNiaNetworkDataSourceTest.kt

```diff
@@ -45,46 +45,46 @@ class FakeNiaNetworkDataSourceTest {
     }
 
     @Test
-    fun testDeserializationOfTopics() = runTest(testDispatcher) {
-        assertEquals(
-            /* ktlint-disable max-line-length */
-            NetworkTopic(
-                id = "1",
-                name = "Headlines",
-                shortDescription = "News you'll definitely be interested in",
-                longDescription = "The latest events and announcements from the world of Android development.",
-                url = "",
-                imageUrl = "https://firebasestorage.googleapis.com/v0/b/now-in-android.appspot.com/o/img%2Fic_topic_Headlines.svg?alt=media&token=506faab0-617a-4668-9e63-4a2fb996603f",
-            ),
-            /* ktlint-enable max-line-length */
-            subject.getTopics().first(),
-        )
-    }
+    fun testDeserializationOfTopics() =
+        @Suppress("ktlint:standard:max-line-length")
```

Comment:
nit: Can this be moved on the function scope itself?

```
@Suppress("ktlint:standard:max-line-length")
@Test
fun testDeserializationOfTopics() = runTest(testDispatcher) {
```

Reply:
Updated: d671126f9eb5532206db2c7260e3cc06a978dd77